vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -326,4 +328,29 @@ protected boolean isSelected(T item) {
     public T getEmptyValue() {
         return null;
     }
+
+    /**
+     * Sets the dropdown width.
+     *
+     * @param width
+     *            the new dropdown width. Pass in null to set the dropdown width
+     *            back to the default value.
+     */
+    public void setDropdownWidth(String width) {
```

Comment:
Method should probably have corresponding name `setOverlayWidth`. Same below.
Even though `dropdown` can be more intuitive, part name, css property already contains `overlay`, so we should align it for consistency.

Reply:
Overlay is a technical implementation and an implementation detail so it should not leak into the higher-level API. Also drop-down looks more common, e.g. [swing](https://docs.oracle.com/javase/tutorial/uiswing/components/combobox.html) and [windows forms](https://learn.microsoft.com/en-us/dotnet/desktop/winforms/controls/combobox-control-overview-windows-forms?view=netframeworkdesktop-4.8). However, it's true that [Vaadin docs](https://vaadin.com/docs/latest/components/combo-box) mention overlay, so let's go with the official terminology. Let's keep the dropdown at least in javadoc though.